src/syft/proto/util/json_message_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: proto/util/json_message.proto
-"""Generated protocol buffer code."""
+
```

Comment:
Sorry, this is really annoying but protoc 3.13.0 which just got released seems to include these """Generated protocol buffer code.""" strings again. Can you make sure you have the latest protoc just so we can keep the noise down on these where possible.